zerver/management/commands/reupload_emoji.py

```diff
@@ -0,0 +1,44 @@
+import io
+from typing import Any
+from urllib.parse import urljoin
+
+from django.core.management import CommandError
+from django.core.management.commands import sendtestemail
+
+from zerver.lib.emoji import get_emoji_url
+from zerver.lib.outgoing_http import OutgoingSession
+from zerver.lib.upload import upload_emoji_image
+from zerver.models import RealmEmoji, UserProfile
+
+
+class Command(sendtestemail.Command):
+    def handle(self, *args: Any, **kwargs: str) -> None:
+        session = OutgoingSession(role="reupload_emoji", timeout=3, max_retries=3)
+
+        for realm_emoji in RealmEmoji.objects.order_by("id"):
+            print(f"Processing emoji {realm_emoji.id}")
+            emoji_filename = realm_emoji.file_name
+            emoji_url = get_emoji_url(emoji_filename, realm_emoji.realm_id)
+            if emoji_url.startswith("/user_avatars"):
+                emoji_url = urljoin(realm_emoji.realm.uri, emoji_url)
```

Comment:
Hmm, can we not use library code to do this URL computation?

Reply:
`/` or `/user_avatars` should be the same in terms of what happens, but changed to `/` in case that looks cleaner

Reply:
Yeah, that does seem better.